Makefile

```diff
@@ -9,7 +9,7 @@ export PATH := $(GOPATH)/bin:$(PATH)
 all: build
 build:
 	go fmt ./...
-	go install ./...
+	go get ./...
```

Comment:
In case dependencies get added or changed. Also if someone downloads the git repo and builds, rather than using `go get`. Note, `go get -u ./...` would also update dependencies, but that comes at quite a cost in performance, so we might want to do that in a different target (`rebuild`? or maybe create a new `clean` target?) .....